src/main/java/io/Ui.java

```diff
@@ -0,0 +1,110 @@
+package io;
+
+import java.util.ArrayList;
+import tasks.Deadline;
+import tasks.Event;
+import tasks.Task;
+import tasks.Todo;
+
+/**
+ * The Ui class displays information to the user
+ */
+public class Ui {
+
+
+    private ArrayList<String> printArray;
```

Comment:
The name `printArray` would be more applicable for a method that prints an output array. Perhaps a noun, such as `arrayToBePrinted`, would be a more suitable name for your `ArrayList`?